newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -96,7 +96,7 @@ const FeedbackCard = ({
       console.error(`Unable to update comment: `, error);
       showErrorBox({
         message:
-          i18n._(t`Unable to change resolved status of feedback.`) +
+          i18n._(t`Unable to change read status of feedback.`) +
```

Comment:
"Unable to mark feedbacks as read."?

Reply:
Good idea

Reply:
But no in fact, it is used to mark a feedback as unread to so I prefer having it as is and not have 2 sentences to translate